.github/workflows/update-snapshots.yml

```diff
@@ -96,6 +96,10 @@ jobs:
         run: npm install -g @microsoft/rush@5.112.1
       - name: Install dependencies
         run: rush install --max-install-attempts 3
+      - uses: browser-actions/setup-chrome@v1
```

Comment:
Similar question here: to have 1 job on which other will depend on and use it's output value in `CHROME_PATH`